src/modules/hip/kernel/image_sum.hpp

```diff
@@ -0,0 +1,481 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+#include "reduction.hpp"
+
+// -------------------- Set 0 - Reduction Stage 2 --------------------
+
+__global__ void image_sum_grid_result_tensor(float *srcPtr,
```

Comment:
we need to revisit. Adding all u8 values and putting it into float output will loose precision. Also where is this kernel used. If we are outputing into float, the input has to be converted and normalized to float (0-1.f ) range before summing

Reply:
Changed - added Rpp32u (uint) and Rpp32s (int) sums support